packages/app/src/index.js

```diff
@@ -18,10 +18,12 @@ ReactDOM.render(
         <NominationsDataProvider>
           <SearchResultDataProvider>
             <ActiveNominationProvider>
-              <Route path={['/', '/home']} component={Home} />
               <Route exact path={'/login'} component={Login} />
               <Route exact path={'/searchresults'} component={SearchResults} />
-              <Route path="/nomination/:id" exact component={NominationsPage} />
+              <Route exact path="/nomination/:id" component={NominationsPage} />
+              <Route exact path={['/', '/home']} component={Home} />
+              {/* <Route component={Home} /> */}
```

Comment:
Did we want to comment out this catch all route here?

Reply:
Per discussion, will comment this out right now, will eventually do a 404 Not Found page for unavailable urls